test/SparqlAntlrParserTest.cpp

```diff
@@ -199,8 +199,9 @@ TEST(SparqlExpressionParser, First) {
   auto resultAsExpression = std::move(resultofParse.resultOfParse_);
 
   VariableToColumnMap map;
+  using namespace ad_utility::memory_literals;
   ad_utility::AllocatorWithLimit<Id> alloc{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1000)};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1_kB)};
```

Comment:
also use the `getAllocator()` testing function here.

Reply:
Improving already existing tests should be a separate branch.

Reply:
Done, see [758bbbe](https://github.com/ad-freiburg/qlever/pull/1112/commits/758bbbe549e53fbf83c0fe6d30620026eef3e7a3).